src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -200,6 +200,71 @@ void BLEManagerImpl::sl_ble_event_handling_task(void * args)
     }
 }
 
+uint32_t rsi_ble_add_matter_service(void)
+{
+    uuid_t custom_service                                   = { RSI_BLE_MATTER_CUSTOM_SERVICE_UUID };
+    custom_service.size                                     = RSI_BLE_MATTER_CUSTOM_SERVICE_SIZE;
+    custom_service.val.val16                                = RSI_BLE_MATTER_CUSTOM_SERVICE_VALUE_16;
+    uint8_t data[RSI_BLE_MATTER_CUSTOM_SERVICE_DATA_LENGTH] = { RSI_BLE_MATTER_CUSTOM_SERVICE_DATA };
+
+    static const uuid_t custom_characteristic_RX = {
+        .size     = RSI_BLE_CUSTOM_CHARACTERISTIC_RX_SIZE,
+        .reserved = { RSI_BLE_CUSTOM_CHARACTERISTIC_RX_RESERVED },
+        .val      = { .val128 = { .data1 = { RSI_BLE_CUSTOM_CHARACTERISTIC_RX_VALUE_128_DATA_1 },
+                                  .data2 = { RSI_BLE_CUSTOM_CHARACTERISTIC_RX_VALUE_128_DATA_2 },
+                                  .data3 = { RSI_BLE_CUSTOM_CHARACTERISTIC_RX_VALUE_128_DATA_3 },
+                                  .data4 = { RSI_BLE_CUSTOM_CHARACTERISTIC_RX_VALUE_128_DATA_4 } } }
+    };
```

Comment:
All these seem to be constants. If they are only used in this function, they can `constexpr` and their names can be adapted to fit the constant naming convention.

Reply:
Done